design/FY2019/AddCoilandWaterheaterChoicesforDesuperheater.md

```diff
@@ -0,0 +1,102 @@
+# Add More Coil/WaterHeater Choices for Coil:WaterHeating:Desuperheater
+
+
+**Yueyue Zhou**
+
+**National Renewable Energy Laboratory**
+
+**April 2, 2019**
+ 
+
+## Justification for New Feature ##
+
+Desuperheater is a widely used new technology making use of waste heat from HVAC system to heat domestic hot water. Desuperheater is available in E+ only supporting 
+a very limited amount of HVAC systems and water heater tanks, while the most widely used occasion for desuperheater: ground source heat pump system is not yet available.
+ Given only 8 valid choices of cooling and refrigeration components and only 1 tank option, it constrained the variations people modeling their hybrid systems and impeded 
+ the promotions of using this new energy efficiency techonology. Users and interface developers are requesting EnergyPlus broaden its capability of modeling desuperheater 
+ with more coils and tanks options.Some other energy modeling  software tools like Rem/Rate, Ekotrope, Wrightsoft are already capable to model this hybrid GSHP system with 
+ simple assumptions and methods. Since EnergyPlus already supports some systems, it would be more straight-forward to add more enumarations to EnergyPlus.
+
+
+## Overview ##
+
+EnergyPlus would be modified to add more enumarations in Coil:WaterHeating:Desuperheater object's heating source object type and tank object type fields.
+
+## Approach ##
+
+EnergyPlus already handles HVAC/water heating hybrid systems for several coil types. The workflow is straight-forward to add the same structure as those systems. The steps to new enumarations are given below.
+
+1. IDD file modification: See below the entire list of **Coil:WaterHeating:Desuperheater** modified field:
+
+```
+Coil:WaterHeating:Desuperheater,
+  A7 , \field Tank Object Type
+       \type choice
+       \key WaterHeater:Mixed
+       [Add]\key WaterHeater:Stratified
+       \default WaterHeater:Mixed
+       \note Specify the type of water heater tank used by this desuperheater water heating coil.
+	   
+  A9 , \field Heating Source Object Type
+       \required-field
+       \type choice
+       \key Coil:Cooling:DX:SingleSpeed
+       \key Coil:Cooling:DX:TwoSpeed
+       \key Coil:Cooling:DX:TwoStageWithHumidityControlMode
+       \key Coil:Cooling:DX:VariableSpeed
+       [Add]\key Coil:Cooling:WaterToAirHeatPump:EquationFit
+       [Add]\key Coil:Cooling:WaterToAirHeatPump:ParameterEstimation
```

Comment:
OK, now there is some importance as to how much heat is rejected to the water-side of these 2 coils. In the original model, the remaining amount of waste heat was simply rejected to air and did not impact the simulation (except for the extra sub-cooling of the refrigerant which would increase the amount of air-side capacity and was neglected). Now the desuperheater will be reducing the amount of heat transfer to the water-side of this coil which will impact the plant. Maybe it is as simple as subtracting the heat recovered from the heat transfer to the water-side. Just something to think about moving forward. Also, see if you can understand why both Refrigeration:CompressorRack and Refrigeration:Condenser* are included in these lists, it doesn't make a lot of sense (except that you could recover heat from either place in the refrigerant piping, i.e., leaving the compressor or entering the condenser -- but then that's the same thing since the refrigerant piping is not simulated).